### PR TITLE
Added Release Candidate Pull Request Template

### PR DESCRIPTION
**Pull request by Shinkson47**
> _NB_ : As a meta, this is directly against `main` -  NOT `develop`.

Adds a template for standardising the creation of PR's which finalise releases.

Read, render, and lmk if you have anything you'd like to add anything to it.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/Release Candidate.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+title: 'Release the sprint x candidate '
+assignees: ''
+
+---
+
+<!--
+Release routine checklist:
+ * Capture the `develop` branch as `Candidate-Release/x`. This is the candidate for release.
+ * Open PR, filling appropriate details.
+ * Request review from Gal, and only Gal. He will act as a proxy for bringing fauna's feedback to the repository.
```
Comment (galdchen) [inline .github/PULL_REQUEST_TEMPLATE/Release Candidate.md:13]: Don't embarrass me you silly goose.

**Review by galdchen — APPROVED** [review]
As always, very organized, diligent, and impressive. Looks like you covered all your bases. Approve

File: .github/PULL_REQUEST_TEMPLATE/Release Candidate.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+<!-- See sprint goal in the card -->
+
+# Changelist
+<!-- A brief overview of the changes brought by the sprint -->
+ * Implemented a new feature
+ * Fixed major bug 
+
+# Card list
+*(Log of work completed)*
+<!-- Copy the items in the card, implementing 'closes' or 'closed' on the various issues. -->
```
Comment (atallahhezbor) [inline .github/PULL_REQUEST_TEMPLATE/Release Candidate.md:46]: Do the items in the card necessarily reflect the work completed? Would it be better to rely on the changelog of commits from develop?

**Review by atallahhezbor — COMMENTED** [review]
The above looks good to me! Though I did want to level set on the expected behavior when opening the RC PR (see above)

Also, how will this be restricted to only PRs against main? I'd imagine we don't want general PRs against develop having to delete all the above

**Comment by Shinkson47** [comment]
@atallahhezbor 
> how will this be restricted to only PRs against main?
uwu pwease don't use this against develop 👉 👈 

> we don't want general PRs against develop having to delete all the above
There should be a 'blank' option just link on issues, but for clarity i can add a blank template. It'll be more visible.

> PRs against main should build for Mac and Win?
My bad, I didn't realise the workflow was stripped back to that extent.
I remembered talking about reducing our cloud minutes usage by just building for one against develop, and was under the assumption that there was still a hook for both for against main, I should've checked.

> Do the items in the card necessarily reflect the work completed?
Good point, i'll remove that note but i'll keep the card list so we have a list of all included issues, including what wasn't completed, in the overview.

**Pull request merged**